Navigation/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { StyleSheet, Text, View, ScrollView, Image, StatusBar, FlatList, TouchableOpacity, TouchableNativeFeedback,
+  Button,
+  Platform } from 'react-native';
+import { DefaultTheme, Headline, Paragraph, TextInput,
+  Appbar, AppbarAction, AppbarBackAction, AppbarContent, AppbarHeader, 
+  Card, CardContent,
+  // Button,
+  ListAccordion, Divider,
+  ListSection, withTheme,
+  Toolbar, ToolbarBackAction, ToolbarContent, ToolbarAction, 
+  FAB, DrawerSection, Colors,
+  Provider as PaperProvider } from 'react-native-paper';
+import { createStackNavigator, DrawerActions } from 'react-navigation';
+import { Constants } from 'expo';
+import Slideshow from 'react-native-slideshow';
+import { 
+  SendEmail, 
+  Walkthrough, 
+  ConfirmEmail,
+  NoAccount,
+  CreateProfile,
+  EnableNotifications 
+} from '../Screens';
+
+export default createStackNavigator(
+  {
+    ConfirmEmail: {
+      screen: ConfirmEmail,
+    },
+    Walkthrough: {
+      screen: Walkthrough,
+    }, 
+    SendEmail: {
+      screen: SendEmail,  
+    }, 
+    NoAccount: {
+      screen: NoAccount,
+    },
+    CreateProfile: {
+      screen: CreateProfile,
+    }, 
+    EnableNotifications: {
+      screen:EnableNotifications,
+    },
+  },
+  {
+    initialRouteName:'SendEmail'
```

Comment:
2 things here.
1. Not sure if you know, but if you move `SendEmail` to the top, then it will render that one first by default. This solution is still fine by me, just a lil tip.
2. Unused imports (Idk why these bother me... I'm sorry lol)
3. index.js should really just say something like `export { default as AppNavigator } from "./Navigation"` and there should be a file called `Navigation.js` here.